src/Kernel32.Desktop/Kernel32.cs

```diff
@@ -898,7 +898,7 @@ public static partial class Kernel32
         ///     </para>
         /// </param>
         /// <returns>
-        ///     If the function succeeds, the return value is a nonzero value.
+        ///     If the function succeeds, the return value is a handle to the loaded module.
         ///     <para>
         ///         If the function fails, the return value is zero. To get extended error information, call
```

Comment:
Ah yes, thanks. "handle to the loaded module" is the wording of MSDN for `LoadLibraryEx` and I found it a lot better than the one they use in `LoadLibrary` so that's why I updated that one as well.